pyomo/network/port.py

```diff
@@ -448,7 +458,7 @@ def Equality(port, name, index_set):
             Port._add_equality_constraint(arc, name, index_set)
 
     @staticmethod
-    def Extensive(port, name, index_set, include_splitfrac=False,
+    def Extensive(port, name, index_set, include_splitfrac=None,
```

Comment:
You might want to update the documentation below to correspond with the change in function API.

Reply:
Done.